src/gtfx_flex_validator.py

```diff
@@ -34,9 +35,7 @@ def process(message) -> None:
                 gtfs_upload_message = QueueMessage.to_dict(message)
                 upload_msg = FileUploadMsg.from_dict(gtfs_upload_message)
                 logger.info(upload_msg)
-                # upload_message = GTFSFlexUpload.data_from(gtfs_upload_message)
-                process_thread = threading.Thread(target=self.process_message,args=[upload_msg])
-                process_thread.start()
+                self.process_message(upload_msg)
```

Comment:
Can we add try catch block to process_message method. In case failure we need to respond to queue about failure